dependencies.md

```diff
@@ -1,4 +1,20 @@
-# Composer log
+# System Requirements
+
+* A webserver like Apache or Nginx
+* PHP from 5.4
```

Comment:
@alexislefebvre
As said here https://github.com/Victoire/victoire/commit/f4c8e427e551b0a1501bb23e654bb1f12755bca0#commitcomment-20861764
we can wait until Victoire is Symfony 3 ready to change PHP minimum version. If you're ok with that, can you approve this PR ? Thx !